submitit/core/test_core.py

```diff
@@ -200,6 +201,17 @@ def test_fake_executor_batch(tmp_path: Path) -> None:
                 job = executor.submit(_three_time, 8)
 
 
+def test_unpickling_watcher_registration(tmp_path: Path) -> None:
+    executor = FakeExecutor(folder=tmp_path)
+    job = executor.submit(_three_time, 4)
+    job._job_id = "007"
+    assert job.watcher._registered == {"12"}  # still holds the old job id
```

Comment:
Could we use a variable name instead of "12" ? 
I understand what the test is doing but it's a bit convoluted and I'd like to make it easier to read.

Reply:
In my opinion "12" has the same status as a variable name :D but that's very personal :D

Reply:
I was more thinking of `original_job_id = job._job_id`
I wrote a small commit that I prefer.